client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NumberInputCell.tsx

```diff
@@ -14,9 +14,9 @@ export const NumberInputCell = <T extends RecordWithId>({
   rowIndex,
   columnIndex,
   isDisabled = false,
-  // Make the default min=1 as this is the typical implementation
+  // Make the default min=0 as this is the typical implementation
   // in Data Tables
-  min = 1,
+  min = 0,
```

Comment:
I think this is an interim solution. Only min=0 works for this component, otherwise can't clear the value when trying to change it (e.g. can't backspace `1` to change from `1` to `2`)

This is because of the `onChange` method below - it resets the input value to the `min` if it receives undefined (i.e. empty input)

We do this because we then call the `updater`, which updates row data. Most places have other columns in the row that are calculated (e.g. you're entering # packs, which affects calculated # units field), so we can't store undefined against the row value (as # packs = undefined would result in # units = NaN).

It would be nice to store a safe default (0 or min) against the row, but have the input itself be clearable, at least while in focus. Note: for reference, this is what was done in the PackVariantEntryCell, which I think works well.

However, we can't currently do this in this component, because we use buffer state instead of typical react state to store the current input value. This is done to handle potential changes to the input value made externally to the component (e.g. changing "issue in packs of" changes the number of packs elsewhere).

All this to say, I think actually having min values in this component/in tables is a bigger refactor, not really hot fix worthy. 

Unless there is a solution I'm not seeing?